docs/recipes/pantheon.md

```diff
@@ -45,7 +45,7 @@ git clone https://github.com/kalabox/lando.git
 cd lando/examples/pantheon
 
 # Clone drops-7 into a nested docroot
-git clone git@github.com:pantheon-systems/drops-7.git web
+git clone https://github.com/pantheon-systems/drops-7.git web
```

Comment:
I thought it will be better to have the simplest recipe possible for beginners, with fewer requirements.
This way - users don't need to get a pantheon ssh pubkey before they can see the recipe working.